### PR TITLE
Make the protocol and messages public

### PR DESCRIPTION
**Pull request by davidfowl**
- More API review feedback

### CODE REVIEW

File: src/Microsoft.AspNetCore.SignalR.Common/CloseMessage.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.AspNetCore.SignalR.Internal.Protocol
+namespace Microsoft.AspNetCore.SignalR
```
Comment (JamesNK) [inline src/Microsoft.AspNetCore.SignalR.Common/CloseMessage.cs]: Hmmm, I'm not sure about putting them in the main namespace. How often will devs interact with these classes?

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Common/CloseMessage.cs]: You think a .Protocol namespace?

Comment (JamesNK) [reply src/Microsoft.AspNetCore.SignalR.Common/CloseMessage.cs]: Yeah

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Common/CloseMessage.cs]: Will do.

**Pull request merged**